### PR TITLE
Edit testpost

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: src/site/content/en/blog/test-post/index.md [hunk 1]
```diff
@@ -513,3 +601,74 @@ porta dolor erat, vel molestie dolor posuere in. Nam vel elementum augue. Nam
 quis enim blandit, posuere justo dignissim, scelerisque diam. Fusce aliquet urna
 ac blandit ullamcorper. Proin et semper nibh, sit amet imperdiet velit. Morbi at
 quam sem.
+
+{% Details %}
```
Comment (MichaelSolati) [inline src/site/content/en/blog/test-post/index.md:605]: The margin and border between these details looks off 
image

File: src/site/content/en/blog/test-post/index.md [hunk 2]
```diff
@@ -513,3 +601,74 @@ porta dolor erat, vel molestie dolor posuere in. Nam vel elementum augue. Nam
...
+{% Instruction 'reload-app' %}
+
+If users are reloading a traditional web page, use the `reload-page` argument:
+
+{% Instruction 'reload-page' %}
+
+If users are reloading a page for the purpose of profiling,
+use the `start-profiling` argument:
+
+{% Instruction 'start-profiling' %}
```
Comment (MichaelSolati) [inline src/site/content/en/blog/test-post/index.md:666]: One of the `li` is off. Probably should set `li` to use `display: inline;`

image

**Review by MichaelSolati — APPROVED** [review]

**Review by MichaelSolati — CHANGES_REQUESTED** [review]
See comments

File: src/site/content/en/blog/test-post/index.md (hunk 1 above)
Comment (devnook) [reply src/site/content/en/blog/test-post/index.md:605]: Yes, this is why these elements were added to test post, to catch all regressions we have elswhere in the code.

File: src/site/content/en/blog/test-post/index.md (hunk 2 above)
Comment (devnook) [reply src/site/content/en/blog/test-post/index.md:666]: Yes, this is why these elements were added to test post, to catch all regressions we have elswhere in the code.

**Pull request merged**